bls-chart/templates/deployment.yaml

```diff
@@ -27,20 +28,44 @@ spec:
         tags.datadoghq.com/service: {{ .Values.name }}
         tags.datadoghq.com/project: {{ .Values.project }}
     spec:
-      serviceAccountName: {{ .Values.name }}
+      serviceAccountName: {{ .Values.serviceAccount.name }}
       containers:
         - name: {{ .Values.name }}
           image: "{{ .Values.image.repository }}:{{ $imageTag }}"
```

Comment:
Shouldn't we also have the `imagePullPolicy` as parameter (I would set it per default to `Always` or `IfNotPresent`)